public/js/main-template.js

```diff
@@ -12,93 +12,122 @@ $(document).ready(function () {
     asideBarRight = document.getElementById("AsideBarRight");
     fieldContainer = document.getElementById("field-container");
     installcontainer = document.getElementById("install-container");
+   if (document.getElementById("pannelone")) {

```

Comment:
id can be "pannelone" camelCase like "pannelOne" or with "-" like "pannel-one" 
```suggestion
   if (document.getElementById("pannel-one")) {

```